### PR TITLE
#83 add web prettier config

### LINKED ISSUE #83: Add prettier config to web project
Add prettier to `web` and update files for consistent formatting.

## Expected Behaviour

Source files with consistent formatting.

## Current Behaviour

Source files are inconsistently formatted.

## Possible Solution

Add prettier.

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #83.

Branches of #88 (review after #88 is merged).

## Description

- Add  `prettier` to `web` project.
- Prettify `web` project files (config, src, stories), etc.

### Documentation:

Tick below if one of the following applies:

1. Documentation has been added to cover changes introduced by this PR.
2. This PR requires no changes to documentation.

- [ ] Documentation OK.

### Tests:

Tick below if one of the following applies:

1. Tests have been added to cover changes introduced by this PR.
2. This PR requires no new tests.

- [x] Tests OK.

### CODE REVIEW

File: web/src/components/atoms/core/index.tsx [hunk 1]
```diff
@@ -7,4 +7,4 @@ export { Table, TableProps } from './Table';
 export { TableBody, TableBodyProps } from './TableBody';
```
Comment (mark-prins) [inline web/src/components/atoms/core/index.tsx:7]: single quotes? all the other files are double quotes?

File: web/src/components/molecules/core/ClearButton.tsx [hunk 2]
```diff
@@ -1,8 +1,16 @@
+<<<<<<< HEAD:web/src/components/atoms/core/ClearButton.tsx
```
Comment (mark-prins) [inline web/src/components/molecules/core/ClearButton.tsx:1]: there was no mention of the merge conflict in the PR?

File: web/src/components/molecules/core/SearchBar.tsx [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+import { ClearIcon, SearchButton, Container } from "../../atoms";
+import { InputField } from "../../molecules";
+import {
+  OnChange,
+  InputChangeElement,
+  OnClick,
+  ButtonClickElement,
+} from "../../../types";
+import { flexStyle, flexRowStyle } from "../../../styles";
+=======
```
Comment (mark-prins) [inline web/src/components/molecules/core/SearchBar.tsx:13]: another conflicted file

File: web/.prettierrc.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mark-prins) [inline web/.prettierrc.json:1]: I'm struggling with prettier's preference for double quoting everything. It's been 10y of single quotes and I'm finding the the change jarring. 
You don't mind the double quotes?

File: web/src/components/molecules/core/SearchBar.tsx (hunk 3 above)
Comment (wlthomson) [reply web/src/components/molecules/core/SearchBar.tsx:13]: Oops, these are fixed in #95, but will fix them here, too!

File: web/.prettierrc.json (hunk 4 above)
Comment (katherine-sussol) [reply web/.prettierrc.json:1]: I think with the nx changes the whole repo will share the same .prettierrc file (unless explicitly overridden), which may end up modifying these files back to the single quotes (oops, wasn't trying to create extra churn but as the login component PR is based on this one it's probably still better to merge?).

File: web/src/components/molecules/core/ClearButton.tsx (hunk 2 above)
Comment (mark-prins) [reply web/src/components/molecules/core/ClearButton.tsx]: resolved.

**Pull request merged**